test/suites/consolidation/suite_test.go

```diff
@@ -560,7 +560,14 @@ var _ = Describe("Consolidation", func() {
 						},
 					},
 					ResourceRequirements: v1.ResourceRequirements{
-						Requests: v1.ResourceList{v1.ResourceCPU: resource.MustParse("1.5")},
+						Requests: v1.ResourceList{
+							v1.ResourceCPU: func() resource.Quantity {
+								dsOverhead := env.GetDaemonSetOverhead(nodePool)
+								base := lo.ToPtr(resource.MustParse("1800m"))
+								base.Sub(*dsOverhead.Cpu())
```

Comment:
Why is this only for CPU? shouldn't we do this for all resources?

Reply:
Since we're only specifying CPU requests for the pods we're scheduling that should be the only resource under contention. Theoretically daemonset memory requests could grow large enough that they no longer fit on the nodes we're selecting on, but we're nowhere near that memory threshold atm.